packages/framework/src/Model/Customer/Exception/CustomerUserNotFoundException.php

```diff
@@ -4,6 +4,6 @@
 
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
-class UserNotFoundException extends NotFoundHttpException implements CustomerException
+class CustomerUserNotFoundException extends NotFoundHttpException implements CustomerUserException
```

Comment:
I think, that this changes should be committed in previous commit "renamed User* objects to CustomerUser*"